### PR TITLE
Skip fp64 tests: Ngraph

### PR DESCRIPTION
**Pull request by mozga-intel**
No fp64 check for mkldnn op test for now, test=develop #22217

### CODE REVIEW

**Comment by baojun-nervana** [comment]
Can you add `test=develop` in commit note to trigger CI?

File: python/paddle/fluid/tests/unittests/op_test.py [hunk 1]
```diff
@@ -229,6 +232,7 @@ def is_mkldnn_op_test():
             if cls.dtype in [np.float32, np.float64] \
                 and cls.op_type not in op_accuracy_white_list.NO_FP64_CHECK_GRAD_OP_LIST \
                 and not hasattr(cls, 'exist_fp64_check_grad') \
+                and is_ngraph_op_test() \
```
Comment (juncaipeng) [inline python/paddle/fluid/tests/unittests/op_test.py:235]: If it is not ngraph op test,  raise an error.   So `and not  is_ngraph_op_test() \` ?

File: python/paddle/fluid/tests/unittests/op_test.py [hunk 2]
```diff
@@ -1190,6 +1196,10 @@ def check_output(self,
             (hasattr(self, "attrs") and "use_mkldnn" in self.attrs and \
                     self.attrs["use_mkldnn"] == True):
             self.__class__.use_mkldnn = True
+        if (hasattr(self, "use_ngraph") and self.use_ngraph == True) or \
+            (hasattr(self, "attrs") and "use_ngraph" in self.attrs and \
+                    self.attrs["use_ngraph"] == True):
+            self.__class__.use_ngraph = True
```
Comment (juncaipeng) [inline python/paddle/fluid/tests/unittests/op_test.py:1202]: Do all ngraph op tests have use_ngraph in self or self.attrs? 

Just like skipping the mkldnn op test,  is it necessary to save `self.__class__.use_ngraph` in _append_ops() function？

**Comment by baojun-nervana** [comment]
addressed in other PR.

**Pull request closed without merging**